MediaBrowser.Controller/Drawing/IImageEncoder.cs

```diff
@@ -35,17 +36,22 @@ public interface IImageEncoder
         /// <value><c>true</c> if [supports image encoding]; otherwise, <c>false</c>.</value>
         bool SupportsImageEncoding { get; }
 
+        /// <summary>
+        /// Get the dimensions of an image from the filesystem.
+        /// </summary>
+        /// <param name="path">The filepath of the image.</param>
+        /// <returns>The image dimensions.</returns>
         ImageDimensions GetImageSize(string path);
 
         /// <summary>
-        /// Encodes the image.
+        /// Encode an image.
         /// </summary>
         string EncodeImage(string inputPath, DateTime dateModified, string outputPath, bool autoOrient, ImageOrientation? orientation, int quality, ImageProcessingOptions options, ImageFormat outputFormat);
```

Comment:
Doesn't the analyzer complain about missing param and return comments?

Reply:
This class is from a different project `MediaBrowser.Controller.Drawing`, which doesn't have any of the analyzers turned on. I navigated to this class while working on this and didn't realize it was in a different project, so I can revert this change if necessary.